hw/mcu/dialog/da14699/include/mcu/mcu.h

```diff
@@ -114,8 +114,10 @@ typedef enum {
 #define MCU_GPIO_MODE_OUTPUT                0x300    /**< GPIO as an output */
 #define MCU_GPIO_MODE_OUTPUT_OPEN_DRAIN     0x700    /**< GPIO as an open-drain output */
 
+#define MCU_GPIO_PORT0_PIN_COUNT            32
 #define MCU_GPIO_PORT0(pin)		((0 * 32) + (pin))
```

Comment:
Seems like a great idea to have the pin count for ports, should `MCU_GPIO_PORT0/PORT1` use this pin count instead of 32 ?